lib/tasks/example_data.rake

```diff
@@ -40,64 +37,41 @@ namespace :example_data do
 
       next unless persona_attributes[:provider]
 
+      # Create the provider for that persona
       provider = Provider.find_or_create_by!(
         name: persona_attributes[:provider],
         dttp_id: SecureRandom.uuid,
         code: Faker::Alphanumeric.alphanumeric(number: 3).upcase,
       )
-      FactoryBot.create_list(:course, rand(30..70), accredited_body_code: provider.code, route: "provider_led_postgrad")
       persona.update!(provider: provider)
 
-      rand(50...100).times do
-        traits = trait_combinations.sample
-
-        created_at = Faker::Date.between(from: 100.days.ago, to: 50.days.ago)
-        submitted_for_trn_at = nil
-        trn = nil
-        progress = {}
+      # For each of the course routes enabled...
+      enabled_course_routes.each do |route|
+        # Create some courses for that provider with some subjects
+        FactoryBot.build_list(:course, rand(10..70), accredited_body_code: provider.code, route: route) { |course|
```

Comment:
could we make the multiline block here a `do end` instead of the curly braces

Reply:
done!